File: santanderbootcamp/bin/edu/gustavo/desafiosdeprojeto/CalculoSalarial.cs
using System;

public class CalculoSalarial
{
    public static void Main()
    { 
        float valorSalario = float.Parse(Console.ReadLine());
        float valorBeneficios = float.Parse(Console.ReadLine());

        float valorImposto = 0;

            // de R$0,00 a R$1100.00
        if (valorSalario >=0 && valorSalario <= 1100) {
            valorImposto = 0.05F * valorSalario;
            // de R$1100.01 a R$2500.00
        }else if (valorSalario >=1100.01 && valorSalario <= 2500.00) {
            valorImposto = 0.10F * valorSalario;
            // acima de R$2500.00    
        } else { 
            valorImposto = 0.15F * valorSalario;
        }

        float saida = valorSalario - valorImposto + valorBeneficios;
        Console.WriteLine(saida.ToString("0.00"));    
    }
}

File: santanderbootcamp/bin/edu/gustavo/desafiosdeprojeto/CalculoSalarial.js
const ValorSalario = parseFloat(gets()); // const pois não mudará!
const valorBeneficios = parseFloat(gets());

const valorImposto = calcularImposto(ValorSalario);
const saida = ValorSalario - valorImposto + valorBeneficios;
print(saida.toFixed(2));

function calcularImposto(salario) {
    let aliquota; // let pois o valor mudará!
    if (salario >= 0 && salario <= 1100) {
        aliquota = 0.05;
    }else if (salario >=1100.01 && salario <= 2500.00) {
        aliquota = 0.10 * salario;
        // acima de R$2500.00    
    } else { 
        aliquota = 0.15 * salario;
    }

    return aliquota * salario;

}